uitools/cpp/Esri/ArcGISRuntime/Toolkit/UtilityNetworkFunctionTraceResultsModel.h

```diff
@@ -19,12 +19,13 @@
 
 #include <QAbstractListModel>
 
+#include "UtilityNetworkFunctionTraceResult.h"
```

Comment:
I think this can be fwd declared

Reply:
Alas, the header isn't declaring a pointer type, so it needs the actual header

Reply:
i think you only use it in a qlist or by const ref so it can be fwd declared